Sources/Core/API/Shadow.swift

```diff
@@ -0,0 +1,34 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+
+/// Structure used to describe a shadow that should be rendered for an object
+public struct Shadow {
+    /// The radius of the shadow, that is, to what distance it should be blurred
+    public var radius: Metric
+    /// The opacity of the shadow (between 0 - 1)
+    public var opacity: Metric
+    /// The color of the shadow (default = .black)
+    public var color: Color
+    /// The offset of the shadow, based on the object's center point
+    public var offset: Point
+    /// Any path to draw the shadow using. Specifying this may increase performance.
```

Comment:
What does "increase performance" mean? In another comment you said using shadows may "impact performance", I think that is more clear; "increase performance" sounds like it might make performance better, but I don't think that's what you meant here, but I don't really know how it being `nil` is handled, so maybe it does improve performance. If so, maybe "improve performance" would be more clear?

Reply:
You're totally right @aranasaurus - nice catch! 😄 It should be "impact performance" 👍 Will fix!

Reply:
@JohnSundell I thought specifying a path for a `CALayer`'s shadow would in fact increase performance. I heard they mention it on some WWDC presentation 🤔

Reply:
Double brainfart 😅 You're totally right @insidegui, what it **really** should say is "improve performance". Making that change now 👍